thoth/storages/graph/postgres.py

```diff
@@ -4164,11 +4164,12 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
         os_name = document["result"]["operating-system"]["name"]
         os_version = document["result"]["operating-system"]["version_id"]
-        cuda_version = document["result"]["cuda-version"]["nvcc_version"]
-        if cuda_version != document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]:
+        cuda_version = document["result"].get("cuda-version", {}).get("nvcc_version", None)
```

Comment:
I'm not sure if this is going to work actually. Did you test it?

Reply:
@pacospace I didn't test, syncing a document, but I tested in python and it works.